tests/linalg_test.py

```diff
@@ -1619,6 +1619,39 @@ def testToeplitzConstructionWithKnownCases(self):
       [2, 1, 4, 5],
       [3, 2, 1, 4]], dtype=np.float32))
 
+  @jtu.sample_product(
+    size=[1, 2, 3, 10, 100],
+    dtype=float_types + complex_types)
+  def testNullSpace(self, size, dtype):
+    rng = jtu.rand_default(self.rng())
+    shape = (2, size)
```

Comment:
Can we iterate over various shapes rather than hard-coding them?

Reply:
I was trying to keep the structure of the original scipy test here, do you think it makes sense for the potential readers?
Otherwise I will just iterate ofc

Reply:
As an example, see this test: https://github.com/google/jax/blob/64145393b2c65bb011ca10d41314a8a06c025cb7/tests/linalg_test.py#L803-L823

We don't test here that the output has the properties expected of an inverse. We use scipy as the ground truth, and check against that.

Reply:
sure will update!

Reply:
done